scout/adapter/mongo/case.py

```diff
@@ -494,21 +487,20 @@ def _conditional_set_query_value(query, condition, set_key, set_value):
 
         if verification_pending:  # Filter for cases with Sanger verification pending
             sanger_pending_cases = self.verification_missing_cases(owner)
-            self._update_case_id_query(query, sanger_pending_cases)
+            self.update_case_id_query(query, sanger_pending_cases)
 
         if has_clinvar_submission:
             clinvar_subm_cases = self.clinvar_cases(collaborator or owner)
-            self._update_case_id_query(query, clinvar_subm_cases)
+            self.update_case_id_query(query, clinvar_subm_cases)
 
         if has_rna_data:
             cases_with_rna = self.rna_cases(collaborator or owner)
-            self._update_case_id_query(query, cases_with_rna)
+            self.update_case_id_query(query, cases_with_rna)
 
         if yield_query:
             return query
 
-        if order:
```

Comment:
Then order is used here, just to say if the array has values use the following projection:

'analysis_date': 1, 'assignees': 1, 'beacon': 1, 'case_id': 1, 'display_name': 1, 'individuals': 1, 'is_rerun': 1, 'is_research': 1, 'mme_submission': 1, 'owner': 1, 'panels': 1, 'status': 1, 'track': 1, 'vcf_files': 1}

I don't think the order thing adds anything by itself in here?

Reply:
Hm, I'm probably slow today, but at least I'm looking.. 😆 It returns them in the order given, without sorting, and the intention appears to be that they should appear in the same order as the "case_id: $in" operation gave them previously. Are you saying this doesn't work anyway, or that the returned similarity values are the same so it doesn't work? 😊